airbyte-ci/connectors/base_images/README.md

```diff
@@ -39,6 +39,21 @@ RUN mkdir -p 755 /usr/share/nltk_data
 
 
 
+### Example for `airbyte/java-connector-base`:
+```dockerfile
+FROM docker.io/amazoncorretto:21-al2023@sha256:5454cb606e803fce56861fdbc9eab365eaa2ab4f357ceb8c1d56f4f8c8a7bc33
+RUN yum update -y --security
+RUN yum install -y tar openssl findutils
```

Comment:
I know this is just the example, but looking into the code further down in this PR, it looks like we're following what this example does.  

We want to limit the number of `RUN` calls in our image creation process. Each `RUN`will create a new layer and further increase the size of our base image.

You can see on the [platform side how we've collapsed multiple `RUN` calls into a single call](https://github.com/airbytehq/airbyte-platform/blob/e047a70391a47ac321b83aae391f2b9583351cd7/docker/airbyte-base-java-image/Dockerfile#L7-L20) to reduce our base image size.

Reply:
@colesnodgrass I can definitely bundle these three commands into one. Thank you for the suggestion.

Reply:
Thank you for the tips @colesnodgrass . 50mb saved 👍